### PR TITLE
Add first version of the a11y guide

### PR DESCRIPTION
**Pull request by jgiardino**
This adds the first version of the a11y guide.

Link: https://deploy-preview-196--pf-next.netlify.com/accessibility-guide

Note: additional css is needed to handle how .md files present lists and nested lists.

### CODE REVIEW

**Comment by jgiardino** [comment]
Thanks for the review, @michael-coker! I addressed most of your suggestions. 

> "as well as physical, sensory or learning" - needs an oxford comma

For this one, I left the oxford comma out since this sentence is already comma heavy.

I'm still on the fence regarding the best way to structure items in the **Design and Development Guidelines** section, so if anyone has suggestions, please let me know. I will probably make refinements in future iterations too.

**Comment by andresgalante** [comment]
@jgiardino I think this is great for alpha 1, and we'll probably improve the Design and Development Guidelines section as problems appear on PRs.

there is a conflict on `src/site/workspace.scss` would you like me to help rebasing your branch?

**Comment by jgiardino** [comment]
Thanks, @andresgalante! I just rebased. Let me know if any changes are needed.

File: src/site/pages/accessibility-guide.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Since accessibility is best achieved when it's considered at the beginning, we ask designers and developers who contribute or consume patternfly to understand how accessibility needs can be met. The following is a guide with some techniques on how to design, develop, and test UI to ensure that everyone has a good user experience.
+
+- [Understanding Users’ Needs](#understanding-users-needs)
+- [Checklists](#checklists)
+  - [Design and Development Guidelines](#design-and-development-guidelines)
+      - [What PatternFly Designers and Developers Should Address](#what-patternfly-designers-and-developers-should-address)
+      - [What Product Developers and Designers Should Address](#what-product-developers-and-designers-should-address)
+  - [Testing](#testing)
+  - [Screen reader support](#screen-reader-support)
```
Comment (matthewcarleton) [inline src/site/pages/accessibility-guide.md]: should this be "Screen Reader Support"?

**Comment by matthewcarleton** [comment]
looks great Jenn!

File: src/site/pages/accessibility-guide.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+As with any design, it’s important to understand the user you’re designing for. To develop an inclusive design, it’s important to understand the different needs of [different users](https://a11yproject.com/posts/myth-accessibility-is-blind-people/), and consider the assistive tools and methods they use. The following are some of the ways we can help address these different needs.
+
+### No Vision
+
+Users with no vision rely on screen readers to access web sites and applications. Often, screen reader users will navigate a page by browsing specific elements on the page, like headers, links, or form elements. Using semantic elements and ensuring that these elements are still meaningful when pulled out of context is critical for the usability of the site.
+
+### Low Vision
+
+Users with low vision can have different needs depending on their eye condition. Needs can range from not relying on color to communicate information, color palettes that provide sufficient contrast, and a layout that is responsive when the user zooms in to magnify the contents of the screen.
```
Comment (srambach) [inline src/site/pages/accessibility-guide.md]: "Eye condition" sounds a little odd to me. How about "depending on the nature of their visual impairment." ? Also, the next sentence is complicated - how about something like "Users may have difficulty with color differentiation, blurriness, or lack of vision in central or peripheral areas. These needs mean that interfaces should not rely on color to communicate information, palettes need to have sufficient contrast, and screens should behave well when a screen magnifier is used."

**Comment by srambach** [comment]
Looks great, @jgiardino! Thank you for all the work you've put into it, especially highlighting and linking the key parts of the spec.
(However, I stand with @michael-coker on the need for the Oxford comma)
:-)

File: src/site/pages/accessibility-guide.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Users with poor motor control can use a range of devices to access contents. If these users are using only a keyboard, then they rely on all elements being keyboard accessible, with elements that have focus being highly visible. If these users are using a mouse or touch, then target areas should be large enough to be easy to hit.
+
+### Cognitive
+
+Users who have difficulty processing information benefit from contents that are well-written. They should be clear and concise, and structured so that the contents are easy to consume by scanning the page. Avoid long paragraphs.
+
+Some users fall into just one category above, but it’s possible for a user to fall into more than one, or use tools and devices designed for a user group they don’t fall into. One of the greatest benefits of inclusive design practices is that methods that are designed for a specific user group will often provide benefits to all users.
+
+
+## Checklists
```
Comment (stacymcauliffe) [inline src/site/pages/accessibility-guide.md]: We might try something like this --
Get rid of the Checklists title and make the sections as follows:
-Design and Development Guidelines 
-Accessibility Checklist for PatternFly Designers and Developers
-Accessibility Checklist for All Product Designers and Developers

File: src/site/pages/accessibility-guide.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+# Accessibility Guide
+
+*Please note, this guide is a work in progress and will be updated regularly. We welcome your comments and feedback.*
+
+The goal of software accessibility is to remove barriers to the use of technology so that it can be inclusive of everyone, regardless of physical ability. Accessibility addresses permanent to temporary disabilities and includes a range of visual impairments, as well as physical, sensory or learning challenges, including those that occur normally as adults age. Designing Red Hat software with this broad range of accessibility in mind will empower any user to accomplish their goals with our technology.
+
```
[the remark this answers is not in the document]
Comment (jgiardino) [reply src/site/pages/accessibility-guide.md]: solves the oxford comma dilemma ;-)

File: src/site/pages/accessibility-guide.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  - [Screen Reader Support](#screen-reader-support)
+- [Methods](#methods)
+
+## Understanding Users’ Needs
+
+As with any design, it’s important to understand the user you’re designing for. To develop an inclusive design, it’s important to understand the different needs of [different users](https://a11yproject.com/posts/myth-accessibility-is-blind-people/), and consider the assistive tools and methods they use. The following are some of the ways we can help address these different needs.
+
+### No Vision
+
+Users with no vision rely on screen readers to access web sites and applications. Often, screen reader users will navigate a page by browsing specific elements on the page, like headers, links, or form elements. Using semantic elements and ensuring that these elements are still meaningful when pulled out of context is critical for the usability of the site.
```
[the remark this answers is not in the document]
Comment (jgiardino) [reply src/site/pages/accessibility-guide.md]: How about this for the last statement instead: "Use semantic elements and check that labels are meaningful when pulled out of context."

These are really two separate action items to help facilitate methods used by screen reader users.

File: src/site/pages/accessibility-guide.md (hunk 3 above)
Comment (jgiardino) [reply src/site/pages/accessibility-guide.md]: I've definitely struggled with how to label this second main section. I'd like to think about it more before making any changes now.

File: src/site/pages/accessibility-guide.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (stacymcauliffe) [reply src/site/pages/accessibility-guide.md]: Yeah, this sounds good to me!

**Comment by srambach** [comment]
That change sounds good, @jgiardino

**Comment by stacymcauliffe** [comment]
This looks good to me, Jenn! I dont have any more suggestions.

**Comment by dgutride** [comment]
Addresses https://github.com/patternfly/patternfly-next/issues/107

**Pull request merged**